temp_name_lib/src/star_physics/velocity/pulsation.rs

```diff
@@ -0,0 +1,49 @@
+use crate::joris_math::spherical_harmonics::d_plmcos_dtheta::deriv1_plmcos_dtheta;
```

Comment:
Also here I don't understand the usefulness of the complicated `star_physics/velocity/*.rs` setup.